### PR TITLE
Fix v2.0.0 migration script not migrating remoteTimeout

### PR DESCRIPTION
**Pull request by pmalek-sumo**
###### Description

Fixes https://github.com/SumoLogic/sumologic-kubernetes-collection/issues/1412

### CODE REVIEW

File: tests/upgrade_v2_script/static/prometheus_remote_write_timeout.output.yaml [hunk 1]
```diff
@@ -1,79 +1,58 @@
 kube-prometheus-stack:
   prometheus:
     prometheusSpec:
-      scrapeInterval: 30s
```
Comment (kkujawa-sumo) [inline tests/upgrade_v2_script/static/prometheus_remote_write_timeout.output.yaml:4]: Why this file contains changes not related to remoteTimeout?

Comment (pmalek-sumo) [reply tests/upgrade_v2_script/static/prometheus_remote_write_timeout.output.yaml]: Good catch! Should be fixed now.

Comment (kkujawa-sumo) [reply tests/upgrade_v2_script/static/prometheus_remote_write_timeout.output.yaml]: It seems that now changes are related only to remoteTimeout but it different order. May we avoid reordering? I'll be easier to review.

Comment (pmalek-sumo) [reply tests/upgrade_v2_script/static/prometheus_remote_write_timeout.output.yaml]: I guess the reason for this is how `yq` approaches yaml maps/hashmaps/? 🤷

**Review by perk-sumo — APPROVED** [review]

**Pull request merged**